docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -3,7 +3,10 @@ sidebar_position: 4
 ---
 # Designer configuration
 
-The default Designer configuration is defined in [defaultDesignerConfig.conf](https://github.com/TouK/nussknacker/blob/staging/designer/server/src/main/resources/defaultDesignerConfig.conf).
+Designer configuration area contains configs for web application interface, security, various UI settings, database and more. Check [configuration areas](./#configuration-areas) to understand where Designer configuration should be
```

Comment:
`(./#configuration-areas)` is it correct link?

Reply:
corrected to (./Common.md#configuration-area) as  it is [here](https://github.com/TouK/nussknacker/blob/staging/docs/installation_configuration_guide/DeploymentManagerConfiguration.md?plain=1#L9). 

Originally I used stale commit ....